content/source/docs/enterprise/install/installer.html.md

```diff
@@ -192,6 +192,7 @@ If your instance can access the Internet, use the Online install mode.
 1. From a shell on your instance:
     * To execute the installer directly, run `curl https://install.terraform.io/ptfe/stable | sudo bash`
     * To inspect the script locally before running, run `curl https://install.terraform.io/ptfe/stable > install.sh` and, once you are satisfied with the script's content, execute it with `sudo bash install.sh`.
+    * To install on RedHat Enterprise Linux where pre-installation of Docker is necessary, run `curl https://install.terraform.io/ptfe/stable > install.sh` and, once you are satisfied with the script's content, execute it with `sudo bash install.sh no-docker`. 
```

Comment:
@larryebaum Technically speaking, letting the installer install Docker CE on RHEL _will_ work, it just won't be officially supported by Red Hat. That is why we have users install Docker from `rhel-extras` or Docker EE.

I think the below suggestion will suffice. Let me know your thoughts.

```suggestion
    * To inspect the script locally before running, run `curl https://install.terraform.io/ptfe/stable > install.sh` and, once you are satisfied with the script's content, execute it with `sudo bash install.sh`. If Docker is already installed on the instance, which is usually the case when running RHEL, the automated installation of Docker can by bypassed by executing the script as `sudo bash install.sh no-docker`.
```

Reply:
@sudomateo Wonder if something recent broke the installer behavior you indicate, as my customer/prospect experience last week differed. The installer fully disrespects that fact that Docker is installed, and overwrites it with a later version that isn't supported and contradicts our docs with state 1.13.1 is **required.** Further, this appears to have caused a variety of other issues (in their PoV) and required a wipe/reinstall. I've validated the experience on a test instance. Perhaps Replicated broke the installer? As we call our RHEL requirements elsewhere, distinct from other steps, I think its appropriate to continue to call out this RHEL step so it's clear and not potentially confusing. If the argument is saving 2 lines of info and being potentially confusing, vs 2 extra lines but being clear and a better customer experience, I'd favor the better customer experience. Zoom for discussion?